infrastructures/infrastructure-gce/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/GCEInfrastructure.java

```diff
@@ -54,7 +54,9 @@ public class GCEInfrastructure extends AbstractAddonInfrastructure {
 
     public static final String INFRASTRUCTURE_TYPE = "google-compute-engine";
 
-    public static final int DEFAULT_RAM = 2048;
+    public static final String DEFAULT_MACHINE_TYPE = "";
```

Comment:
Why not providing a default machineType as done for EC2? 
Is the default ram of 4GB mandatory? Why not keeping 2GB if it is sufficient?

Reply:
we don't provide a default machineType on EC2 infrastructure, only default cores and ram

Reply:
yes you're right Fabien. And even the ram is 4GB as set by Ali here. 
https://github.com/ow2-proactive/nodesource-addons/commit/b972d4347018f388f28ffe88bb514e4afb595eb4